erpnext_documentation/www/docs/user/manual/en/regional/united_arab_emirates/uae-vat-setup.md

```diff
@@ -3,7 +3,7 @@
 
 ### 1. Setting up Tax Registration No for customer, supplier and company
 
-Set tax registation number in the field tax id for the customer, supplier and company
+Set tax registration number in the field tax id for the customer, supplier and company
```

Comment:
Set Tax Registration Number in the field, Tax ID for the Customer, Supplier, and Company